classes/services/queryBuilders/SubmissionListQueryBuilder.inc.php

```diff
@@ -44,6 +44,38 @@ public function filterBySections($sectionIds) {
 	 * @return object Query object
 	 */
 	public function appGet($q) {
+		$primaryLocale = \AppLocale::getPrimaryLocale();
+		$locale = \AppLocale::getLocale();
+
+		$this->columns[] = Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value) AS section_title');
+		$this->columns[] = Capsule::raw('COALESCE(sal.setting_value, sapl.setting_value) AS section_abbrev');
+
+		$q->groupBy(Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value)'));
+		$q->groupBy(Capsule::raw('COALESCE(sal.setting_value, sapl.setting_value)'));
+
+		$q->leftJoin('section_settings as stpl', function($join) use($primaryLocale) {
+			$join->on('s.section_id', '=', Capsule::raw("'stpl.section_id'"));
+			$join->on('stpl.setting_name', '=', Capsule::raw("'section_title'"));
+			$join->on('stpl.locale', '=', Capsule::raw("'{$primaryLocale}'"));
+		});
+
+		$q->leftJoin('section_settings as stl', function($join) use($locale) {
+			$join->on('s.section_id', '=', Capsule::raw("'stl.section_id'"));
+			$join->on('stl.setting_name', '=', Capsule::raw("'section_title'"));
+			$join->on('stl.locale', '=', Capsule::raw("'{$locale}'"));
```

Comment:
Doesn't this defeat string escaping? (Also below)

Reply:
I think this one would work by just changing out `Capsule::raw("'{$locale}'")` for `$locale`.

Reply:
When I do that, I get the following error:

```
DB Error: Unknown column 'en_US' in 'on clause'
```

This comment walks through the debug process on that line and how we ended up there if that gives any clue to what's going on: https://github.com/pkp/ojs/pull/1483#discussion_r134268995